### DISCUSSION TITLE
Create webjar not possible with used license

### DISCUSSION THREAD

**Discussion by jameskennard** [add2cal/add-to-calendar-button#111] [Ideas / Requests]
Hi,

I wanted to create a webjar for this project but it fails to generate because it cannot detect the license (https://www.webjars.org/). Under the hood, it uses another project to detect the license, (https://github.com/jamesward/oss-license-detector). I'm not sure what the change would be to make this work.

Any thoughts?

All the best,

James.

**Comment by jekuer** [comment]
Switching to another license would help 😆 .

Actually, the first version has been licensed under GPL-3. It was changed, since GPL-3 theoretically requires you to make the project, which uses GPL-3 components also GPL-3 licensed. This usually is no problem, but can be blocking for some projects.
This led to the move to MIT, but with the "common clause" (https://commonsclause.com/), basically prohibiting to sell this script.

I will evaluate this again a little bit close. But open for any input and recommendations!
